src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorSingleStream.java

```diff
@@ -35,7 +35,6 @@
 public class RemoteBitrateEstimatorSingleStream
     extends SinglePacketTransformerAdapter
     implements CallStatsObserver,
-               RecurringRunnable,
```

Comment:
I see this as code cleanup because it makes sense to update the estimate when you have data (and not update it when you don't).

This won't run more often than before (at most every 500ms). It may run less often, if we don't receive any RTP packets or if we receive RTP packets less frequently. There are no expensive operations in the updateEstimate method, so I doubt this will have any noticeable impact on performance (either positive or negative), but I don't have any data to support this claim.